data/node-red/.config.json

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
This *must not* be committed, as it's owned by node red and dynamically changed.